packages/bpk-component-datatable/index.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Column } from 'react-virtualized';
+import BpkDataTable from './src/BpkDataTable';
+
+const BpkColumn = Column;
```

Comment:
Oh so we aren't using the column component defined in `BpkColumn.js`, we are using the one from `react-virtualized` directly? is this intentional?

Reply:
Yeah i noticed this last night - just getting my head around it. The `BpkDataTable` code doesn't render them as JSX elements, but calls them as functions (https://github.com/Skyscanner/backpack/pull/370/files#diff-2bda430a0beb77956f2b5269a49be0e0R129) since `react-virtualized`'s `Table` *requires* that it's children are of type `Column`. 

So, we export `react-virtualized` column, but we actually use our own `BpkColumn` in the tests. Going to re-work this stuff to make it a bit more consistent.

Reply:
@jslvtr can you shine some light as to why it's set up this way? Do i have the right idea?

Reply:
Doesn't this conflict with what is below? We aren't using the version we define ourselves

Reply:
Yeah, we need some way of declaring Columns, but we need to use something that'll automatically add `bpk-column` (or whatever the class name is). What about something like this:
```
<BpkDataTable ...props>
  {
    [
      { dataKey: 'name', label: 'Name', width: 100 },
      { dataKey: 'description', label: 'Description', width: 200, flexGrow: 1 },
      { dataKey: 'bla', label: 'Bla', width: 100, disableSort: true },
    ]
  }
</BpkDataTable>
```

with 
```
const columnShape = PropTypes.shape({
  dataKey: PropTypes.string.isRequired,
  label: PropTypes.string.isRequired,
  width: PropTypes.number.isRequired,
  flexGrow: PropTypes.number,
  disableSort: PropTypes.boolean,
});

BpkDataTable.propTypes = {
  ...,
  children: PropTypes.arrayOf(columnShape).isRequired,
  ...,
};
```
?

Reply:
Or would you guys prefer these intermediate data components that define how `react-virtualized` `Column`s should be created (kinda like what we have now, but a bit better organised)?